src/lib.rs

```diff
@@ -255,6 +256,21 @@ impl Chip {
         Line::new(self.inner.clone(), offset)
     }
 
+    /// Get a handle to multiple GPIO line at a given offsets
+    ///
+    /// The group of lines can be manipulated simultaneously.
```

Comment:
Documentation should cover the caveats of this approach (that all pins must be configured the same way when used).  For the casual user of the library this would likely not be immediately apparent.